### PR TITLE
Customer Home: Update heading logic

### LINKED ISSUE #37930: Customer Home: Did not get any confirmation after launching site
#### Steps to reproduce
1. Go through the current onboarding flow, pick "Blog" as the vertical, pick a free Domain/Plan
2. Once you reach Customer Home, Launch your site (I launched it from that big "Launch" button on the right)
3. Be prompted to pick a Domain and a Plan again
4. Once you're back in Customer Home, notice how there isn't any sort of feedback that your site has been launched. If anything, it's more confusing because it's telling you "Your site has been created" when it already _has been_ created

#### What I expected
To be shown some sort of confirmation that my site has been launched.

Earlier today, I launched a different site (different vertical, I know it wasn't "Blog"), and I saw a notice telling me my site has been launched (`global notice` component). There was also another notice (`banner` component) telling me I've completed all my Checklist items. But I didn't see those in this particular case.

#### What happened instead
I was brought back to Customer Home without any confirmation about my site already launched.

#### Browser / OS version
Google Chrome Version 78.0.3904.97 
Mac OS 10.14.2

#### Screenshot / Video
Screenshot 2019-11-25 11 45 08

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* Post signup, if a site is unlaunched or Atomic, the heading will show a "thank you" message for 30 mins (this is the same functionality, but now restricted to unlaunched sites to avoid confusion documented in #37930).
* After completing the launch flow, the heading will show a "congrats" message, using a query arg passed from the launch flow site destination.
* Outside these cases, the heading will show the standard "My Home" text.

**Before**

A site, just launched, with the incorrect header text and no acknowledgement of the launch.

**After**

A site, just launched.

Screen Shot 2019-12-03 at 2 49 46 PM

#### Testing instructions

* Switch to this PR, create a new site through `/start/test-fse`
* You should see a Thank You message in the header; this will persist for 30 minutes after site creation, or until the site is launched, whichever comes first.
* Launch your new site. You should see a "You launched your site!" message in the header, triggered by `?d=launched` in the URL.
* Removing `?d=launched` from the URL should return the Customer Home header to normal.
* My copy for the new congratulations message needs a second set of eyes.

Fixes #37930

### CODE REVIEW

File: client/my-sites/customer-home/main.jsx [hunk 1]
```diff
@@ -185,31 +185,58 @@ class Home extends Component {
...
 						alt=""
 					/>
 					<FormattedHeader
-						headerText={
-							this.props.siteHasPaidPlan
-								? translate( 'Thank you for your purchase!' )
-								: translate( 'Your site has been created!' )
-						}
-						subHeaderText={ this.getChecklistSubHeaderText() }
+						headerText={ translate( 'You launched your site!' ) }
```
Comment (a8ci18n) [inline client/my-sites/customer-home/main.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'You launched your site' )` *ES Score: 11*

**Comment by danhauk** [comment]
This works great.

Not a blocker: I'm not crazy about the subheader `Next, add fresh content to grow your audience.` I'm wondering if we could make it more generalized (e.g. what if it's a business site and I don't have any fresh content to add?)

I'd suggest something like `Make sure you share it with everyone to show it off.` or something?

**Comment by sixhours** [comment]
> Not a blocker: I'm not crazy about the subheader Next, add fresh content to grow your audience. I'm wondering if we could make it more generalized (e.g. what if it's a business site and I don't have any fresh content to add?)

Yeah, very good point. 

> I'd suggest something like Make sure you share it with everyone to show it off. or something?

This could work, and it ties in nicely to any secondary actions we'd add later!

**Comment by danhauk** [comment]
@sixhours I just pushed up the change to the subheader. If you think of something better to put there please do.

**Pull request merged**